test/gloss/test/io.clj

```diff
@@ -0,0 +1,28 @@
+;;   Copyright (c) Zachary Tellman. All rights reserved.
+;;   The use and distribution terms for this software are covered by the
+;;   Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+;;   which can be found in the file epl-v10.html at the root of this distribution.
+;;   By using this software in any fashion, you are agreeing to be bound by
+;;   the terms of this license.
+;;   You must not remove this notice, or any other, from this software.
+
```

Comment:
Please remove this. The license should be covered by the LICENSE file, the epl-v10.html file doesn't exist, and it's not even clear that Zach would be the copyright holder, since you made the file.

Reply:
cool, glad to make my first (tiny) contribution to clojure ecosystem :smile: 
I removed the license notice, I didn't add it, that test file is just copied from https://github.com/clj-commons/gloss/pull/53
But the submitter of that PR is not Zach I assume they just copied the header from the other test files.

Reply:
Yeah, I'm not clear on copyright, so maybe I'm mistaken. But regardless, I've never liked long, irrelevant headers in code files, so... 😁